File: JS_Tasks/correctSentence.js
// Task Description:
// For the input of your function will be given one sentence.
// You have to return its fixed copy in a way so it’s always starts with a capital letter and ends with a dot.
// Pay attention to the fact that not all of the fixes is necessary.
// If a sentence already ends with a dot then adding another one will be a mistake.

function correctSentence(str) {
	// check if str ends with '.'
	str[str.length - 1] === "." ? null : (str += ".");
	// check if str begins with a capital letter
	str[0] === str.toUpperCase()
		? null
		: (str = str[0].toUpperCase() + str.slice(1));
	return str;
}

// Test Cases:
correctSentence("greetings, friends"); // "Greetings, friends."
correctSentence("Greetings, friends"); // "Greetings, friends."
correctSentence("Greetings, friends."); // "Greetings, friends."

File: Python_Tasks/correct_sentence.py
# Task Description:
# For the input of your function will be given one sentence.
# You have to return its fixed copy in a way so it’s always starts with a capital letter and ends with a dot.
# Pay attention to the fact that not all of the fixes is necessary.
# If a sentence already ends with a dot then adding another one will be a mistake.

def correct_sentence(str):
	if str[0] != str[0].upper():
		str = str[0].upper() + str[1:]

	if str[-1] != ".":
		str += "."

	return str

# Test Cases:
correct_sentence("greetings, friends"); # "Greetings, friends."
correct_sentence("Greetings, friends"); # "Greetings, friends."
correct_sentence("Greetings, friends."); # "Greetings, friends."
correct_sentence("hi"); # "Hi."
correct_sentence("welcome to New York"); # "Welcome to New York."